pytest.ini

```diff
@@ -3,3 +3,6 @@
 addopts = --doctest-modules
 minversion = 7.0
 testpaths = tests
+
+# pytest-timeout
+timeout = 60
```

Comment:
could you please add the units as a comment

Reply:
if it is seconds, let's use a value like 5?

Reply:
Yes, a smaller value makes sense. I was thinking of 10 or so. It can also be 5. The only reason I started with 60 is to be conservative and not disrupt the CI too much in case it proved to be too low.

I'll decrease it to 10 then.

Reply:
I would start with 5 and increase if needed